index.js

```diff
@@ -231,3 +256,23 @@ exports.appActivityAudit = function(isStart, sysname, hostname, username) {
   var audit = new AuditMessage(eIdent, [sysParticipant, userParticipant], null, [sourceIdent]);
   return audit.toXML();
 };
+
+exports.auditLogUsedAudit = function(outcome, sysname, hostname, username, userRole, userRoleCode, auditLogURI, objQuery, objDetails) {
+  var eventID = new Code(110101, 'Audit Log Used', 'DCM');
+  var eIdent = new EventIdentification(exports.EVENT_ACTION_READ, new Date(), outcome, eventID, null);
+
+  var sysRoleCode = new Code(110150, 'Application', 'DCM');
+  var sysParticipant = new ActiveParticipant(sysname, '', true, hostname, exports.NET_AP_TYPE_DNS, [sysRoleCode]);
```

Comment:
I think the `userIsRequestor` boolean when using `sysname` should be false here. I noticed this while doing mine. I've changed the others in my branch.